static/src/javascripts/enhanced-vendor.js

```diff
@@ -0,0 +1,26 @@
+define([
+    'bean',
+    'bonzo',
+    'enhancer',
+    'EventEmitter',
+    'fastdom',
+    'fence',
+
+    // We bundle Lodash where necessary, but we want the core to be cached
+    // efficiently, as most modules use the same core
+    'lodash/collections/forEach',
+    'lodash/collections/map',
+    'lodash/collections/reduce',
+
+    'picturefill',
+    'Promise',
+    'qwery',
+
+    'reqwest',
+    'videojs',
+    'videojsads',
+    'videojsembed',
+    'videojsima',
+    'videojspersistvolume',
+    'videojsplaylist'
+], function () {});
```

Comment:
Many of these actually end up in standard because they are transitive dependencies. This means they won't appear in enhanced at all (because we subtract standard from enhanced).

That said, I think it's necessary to keep these dependencies listed because, if we removed them from the dependency graph of standard, they would be bundled in each bootstrap where they are used. Keeping them listed here means, if they are removed from standard, we can be sure they will be instead bundled in `enhanced-vendor` instead of with each bootstrap.